packages/react-components/react-dialog/src/stories/Dialog/index.stories.tsx

```diff
@@ -14,15 +15,23 @@ export { ChangeFocus } from './DialogChangeFocus.stories';
 export { TriggerOutsideDialog } from './DialogTriggerOutsideDialog.stories';
 export { CustomTrigger } from './DialogCustomTrigger.stories';
 export { WithForm } from './DialogWithForm.stories';
+export { TitleCustomAction } from './DialogTitleCustomAction.stories';
+export { TitleNoAction } from './DialogTitleNoAction.stories';
 
 export default {
-  title: 'Preview Components/Dialog/Dialog',
+  title: 'Preview Components/Dialog',
   component: Dialog,
+  subcomponents: {
+    DialogTrigger,
+    DialogSurface,
+    DialogTitle,
+    DialogActions,
+  },
```

Comment:
😮 ❤️

Reply:
Yeah, I was amazed when I found out about this! 🤩